fivetran_provider_async/operators/fivetran.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Dict, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.utils.context import Context
+from fivetran_provider.operators.fivetran import FivetranOperator
+
+from fivetran_provider_async.triggers.fivetran import FivetranTrigger
+
+
+class FivetranOperatorAsync(FivetranOperator):
+    """
+    `FivetranOperatorAsync` submits a Fivetran sync job , and polls for its status on the
+    airflow trigger.`FivetranOperatorAsync` requires that you specify the `connector_id` of
+    the sync job to start. You can find `connector_id` in the Settings page of the connector
+    you configured in the `Fivetran dashboard <https://fivetran.com/dashboard/connectors>`_.
+
+    :param fivetran_conn_id: `Conn ID` of the Connection to be used to configure
+        the hook.
+    :type fivetran_conn_id: str
+    :param connector_id: ID of the Fivetran connector to sync, found on the
+        Connector settings page in the Fivetran Dashboard.
+    :type connector_id: str
+    :param poke_interval: Time in seconds that the job should wait in
+        between each tries
+    :type poke_interval: int
+    :param fivetran_retry_limit: # of retries when encountering API errors
+    :type fivetran_retry_limit: Optional[int]
+    :param fivetran_retry_delay: Time to wait before retrying API request
+    :type fivetran_retry_delay: int
+    """
+
+    def execute(self, context: Dict[str, Any]) -> None:
+        """Start the sync using synchronous hook"""
+        hook = self._get_hook()
+        hook.prep_connector(self.connector_id, self.schedule_type)
+        hook.start_fivetran_sync(self.connector_id)
+
+        """Defer and poll the sync status on the Trigger"""
```

Comment:
```suggestion
        # Defer and poll the sync status on the Trigger
```

Reply:
Done in [a430049](https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/commits/a4300496e12aeba5d6673638067638d8ecdee9cc)